git-link.el

```diff
@@ -166,7 +166,8 @@
     ("github" git-link-github)
     ("bitbucket" git-link-bitbucket)
     ("gitorious" git-link-gitorious)
-    ("gitlab" git-link-gitlab))
+    ("gitlab" git-link-gitlab)
+    ("visualstudio\\|azure" git-link-azure))
```

Comment:
Under what circumstances would one have an azure domain name?

Reply:
When creating a repository from Azure pipelines, such as this:

https://dev.azure.com/r-darwish/_git/testing

Reply:
Oh I see, these are more or less interchangeable:

https://dev.azure.com/s-shaw/_git/Test == https://s-shaw.visualstudio.com/_git/Test